modules/lib/src/main/resources/assets/js/app/event/CollaborationServerEvent.ts

```diff
@@ -7,7 +7,7 @@ import {ClassHelper} from '@enonic/lib-admin-ui/ClassHelper';
 export class CollaborationServerEvent
     extends Event {
 
-    public static EVENT_NAME: string = 'edit.content.collaborators.update';
+    public static EVENT_NAME: string = 'custom.edit.content.collaborators.update';
```

Comment:
Why is it '`custom.edit.content.collaborators.update`' here, but '`edit.content.collaborators.update`' (without `custom.` prefix) in [collaboration.js](https://github.com/enonic/app-contentstudio/pull/4668/files#diff-f19f59e038f6a56f1445ee348ab987e34fc1d24cd9e4b3f7132fa02303fc965bR56)?